src/rezplugins/build_process/local.py

```diff
@@ -414,7 +417,12 @@ def cancel_variant_install():
         # add release info to variant, and install it into package repository
         release_data = self.get_release_data()
         release_data["release_message"] = release_message
+
+        variant = variant.parent.evaluate_directives(context,
+                                                     variant.index)
+
         variant_ = variant.install(release_path, overrides=release_data)
+
```

Comment:
I have also tried to pass the resolved directive data to `variant.install(path, overrides=overrides)`, but won't work.

Reply:
Update:
This actually works, but the unittest is building same package twice, one in `clean=True`, and one not. Which means in second build, there's an existing package and the `overrides` cannot have the precedence to update `requires`.